go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -116,73 +116,73 @@ var MetricResourceDeleteLatency = prometheus.NewHistogram(prometheus.HistogramOp
 // MetricRequeueServiceCount is the number of times a particular service has been requeued.
 var MetricRequeueServiceCount = prometheus.NewCounter(prometheus.CounterOpts{
 	Namespace: MetricOvnkubeNamespace,
-	Subsystem: MetricOvnkubeSubsystemMaster,
+	Subsystem: MetricOvnkubeSubsystemController,
 	Name:      "requeue_service_total",
 	Help:      "A metric that captures the number of times a service is requeued after failing to sync with OVN"},
 )
 
 // MetricSyncServiceCount is the number of times a particular service has been synced.
 var MetricSyncServiceCount = prometheus.NewCounter(prometheus.CounterOpts{
 	Namespace: MetricOvnkubeNamespace,
-	Subsystem: MetricOvnkubeSubsystemMaster,
+	Subsystem: MetricOvnkubeSubsystemController,
 	Name:      "sync_service_total",
 	Help:      "A metric that captures the number of times a service is synced with OVN load balancers"},
 )
 
 // MetricSyncServiceLatency is the time taken to sync a service with the OVN load balancers.
 var MetricSyncServiceLatency = prometheus.NewHistogram(prometheus.HistogramOpts{
 	Namespace: MetricOvnkubeNamespace,
-	Subsystem: MetricOvnkubeSubsystemMaster,
+	Subsystem: MetricOvnkubeSubsystemController,
 	Name:      "sync_service_latency_seconds",
 	Help:      "The latency of syncing a service with the OVN load balancers",
 	Buckets:   prometheus.ExponentialBuckets(.1, 2, 15)},
 )
 
-var MetricMasterReadyDuration = prometheus.NewGauge(prometheus.GaugeOpts{
+var MetricOVNKubeControllerReadyDuration = prometheus.NewGauge(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
-	Subsystem: MetricOvnkubeSubsystemMaster,
+	Subsystem: MetricOvnkubeSubsystemController,
 	Name:      "ready_duration_seconds",
-	Help:      "The duration for the master to get to ready state",
+	Help:      "The duration for the ovnkube-controller to get to ready state",
 })
 
-// MetricMasterSyncDuration is the time taken to complete initial Watch for different resource.
+// MetricOVNKubeControllerSyncDuration is the time taken to complete initial Watch for different resource.
 // Resource name is in the label.
-var MetricMasterSyncDuration = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+var MetricOVNKubeControllerSyncDuration = prometheus.NewGaugeVec(prometheus.GaugeOpts{
 	Namespace: MetricOvnkubeNamespace,
-	Subsystem: MetricOvnkubeSubsystemMaster,
+	Subsystem: MetricOvnkubeSubsystemController,
 	Name:      "sync_duration_seconds",
 	Help:      "The duration to sync and setup all handlers for a given resource"},
 	[]string{
 		"resource_name",
 	})
 
-// MetricMasterLeader identifies whether this instance of ovnkube-master is a leader or not
-var MetricMasterLeader = prometheus.NewGauge(prometheus.GaugeOpts{
+// MetricOVNKubeControllerLeader identifies whether this instance of ovnkube-controller is a leader or not
+var MetricOVNKubeControllerLeader = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
we can remove this right

Reply:
i think we can't remove it because we still support HA in non-IC world?